src/App.css

```diff
@@ -0,0 +1,46 @@
+#root {
+  background: #df4784ce;
+  color: aliceblue;
+}
+
+.App {
+  text-align: center;
+}
```

Comment:
- [ ] You have some design issues at the bottom of the homepage.  Give this a fix.
![Screenshot from 2021-10-14 04-48-43](https://user-images.githubusercontent.com/65501535/137236949-7de8e71a-1099-4f43-a68e-d775cdcac8fc.png)

Reply:
In case that I got no answer and I saw that my styles are good, I did not changes to them.